vignettes/SetSpecies.Rmd

```diff
@@ -0,0 +1,175 @@
+
+The default setting is to model all mosquito species together. 
+```{r}
+simparams$species
+simparams$species_proportions
+```
+
+### Plot species over time
+```{r,fig.align="center", fig.height=5, fig.width=7}
+plot(x = output_all$timestep, y = output_all$Im_All_count,
+     type = "l", col = cols[5], lwd = 2.5,
+     xlab = "Time (days)", ylab = "N mosquitoes",
+       xaxs = "i", yaxs = "i", bty = "l")
+abline(v = sprayingtimesteps, lty = 2, lwd = 2.5, col = "black")
+grid(lty = 2, col = "grey80", lwd = 1)
+legend("topright", box.lty = 0, legend = c("Spraying", "All mosquitoes"), col = c("black", cols[5]), lty = c(2, 1), lwd = 2.5)
+```
+
+
+## IRS with multiple mosquito species
+
+We will run the same model with IRS as above, but this time with 3 species of mosquitoes. Note that now there are 3 columns for the `ls_theta`, `ls_gamma`, `ks_theta`, `ks_gamma`, `ms_theta`, and `ms_gamma` arguments. Each column corresponds to a species of mosquito, and each row corresponds to the timestep when a round of IRS occurred. Here, we are assuming the same values for each species at each IRS round for simplicity, but the matrix could be edited to have different values by species or over time by adding new columns or rows. 
+
+```{r, fig.align="center", fig.height=4, fig.width=6}
+## Setting 3 species of mosquitoes, 25% An. arabiensis, 25% An. funestus, and 50% An. gambiae
+simparams <- set_species(
+  simparams, 
+  species = list(arab_params, fun_params, gamb_params),
+  proportions = c(0.25, 0.25, 0.5)
+)
+
+peak <- peak_season_offset(simparams)
+
+sprayingtimesteps <- c(1, 2) * year + peak - 3 * month # There is a round of IRS in the 1st and second year 3 months prior to peak transmission.
+
+simparams <- set_spraying(
+  simparams,
+  timesteps = sprayingtimesteps,
+  coverages = rep(.8, 2), # # Each round covers 80% of the population 
+  ls_theta = matrix(2.025, nrow=length(sprayingtimesteps), ncol=3), # Matrix of mortality parameters; nrows=length(timesteps), ncols=length(species) 
+  ls_gamma = matrix(-0.009, nrow=length(sprayingtimesteps), ncol=3), # Matrix of mortality parameters per round of IRS and per species
+  ks_theta = matrix(-2.222, nrow=length(sprayingtimesteps), ncol=3), # Matrix of feeding success parameters per round of IRS and per species
+  ks_gamma = matrix(0.008, nrow=length(sprayingtimesteps), ncol=3), # Matrix of feeding success parameters per round of IRS and per species
+  ms_theta = matrix(-1.232, nrow=length(sprayingtimesteps), ncol=3), # Matrix of deterrence parameters per round of IRS and per species
+  ms_gamma = matrix(-0.009, nrow=length(sprayingtimesteps), ncol=3) # Matrix of deterrence parameters per round of IRS and per species
+)
+
+output <- run_simulation(timesteps = sim_length, parameters = simparams)
+```
+
+Here we can see that now there are 3 species of mosquitoes at the proportions we specified in `set_species()`.
+```{r}
+simparams$species
+simparams$species_proportions
+```
+
+  
+### Plot prevalence
```

Comment:
I'm not 100% sure I can see the difference in the impact of IRS that you are trying to demonstrate (these could just be stochastic differences).
It might be more clear if you compare 2 "example" species:
One with the defaults
One with a single species that has a significantly lower `phi_indoors` set.
That might make the comparison more obvious and also show the user that the vectors profiles can be modified?